### PR TITLE
fix: add xx: entries to all additives

### LINKED ISSUE #8918: Non vegan additives should trigger non vegan status
### What
image
https://world.openfoodfacts.org/additive/e542-bone-phosphate
https://world.openfoodfacts.org/product/15440761/bion-3

### Part of
- #5706

### PR DESCRIPTION
**Pull request by stephanegigandet**
### What
- Add an xx: entry to all additives, this helps additive recognition and ingredients parsing when we don't have a translation.
- Fixes #8918 
### Part of
- #7528 
- #5690

### CODE REVIEW

**Review by teolemon — APPROVED** [review]

File: scripts/snippets/add_xx_entry_to_additives.pl [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    }
+=======
+	if ($line =~ /^\w\w:(E(?:[^,]+))/) {
+		$e_number = $1;
+	}
+	if ((defined $e_number) and ($line =~ /^(#|e_number:|wikidata|efsa)/)) {
+		print "xx:$e_number\n";
+		$e_number = undef;
+	}
+>>>>>>> 601aff8ec564a21a04129e5855b17895f6046016
```
Comment (alexgarel) [inline scripts/snippets/add_xx_entry_to_additives.pl:51]: Do you want to keep those ?

```suggestion
    if ($line =~ /^\w\w:(E(?:[^,]+))/) {
        $e_number = $1;
        chomp($e_number);
    }
    if ((defined $e_number) and ($line =~ /^(#|e_number:|wikidata|efsa)/)) {
        print "xx:$e_number\n";
        $e_number = undef;
    }
```

**Review by alexgarel — APPROVED** [review]
LGTM

File: scripts/snippets/add_xx_entry_to_additives.pl [hunk 2]
```diff
@@ -31,13 +31,24 @@
 
 while (my $line = <STDIN>) {
 
+<<<<<<< HEAD
```
Comment (teolemon) [inline scripts/snippets/add_xx_entry_to_additives.pl]: warning ⚠️

**Review by teolemon — CHANGES_REQUESTED** [review]
Merge tag issue

**Review by teolemon — APPROVED** [review]

**Pull request merged**